wpcom-vip-two-factor/sms-provider.php

```diff
@@ -108,7 +107,7 @@ public function generate_and_send_token( $user ) {
 
 		$sms = get_user_meta( $user->ID, self::PHONE_META_KEY, true );
 
-		return \Automattic\VIP\SMS\send_sms( $sms, $message );
+		\Automattic\VIP\SMS\send_sms( $sms, $message );
```

Comment:
Remove this use of the output from "Automattic\VIP\SMS\send_sms"; "Automattic\VIP\SMS\send_sms" doesn't return anything.